spec.html

```diff
@@ -4415,8 +4398,7 @@
             1. Let _desc_ be ? _O_.[[GetOwnProperty]](_key_).
             1. If _desc_ is not *undefined*, then
               1. If _desc_.[[Enumerable]] is *true*, append _key_ to _names_.
-        1. If _O_.[[Enumerate]] is the ordinary object [[Enumerate]] internal method (<emu-xref href="#sec-ordinary-object-internal-methods-and-internal-slots-enumerate"></emu-xref>), then
-          1. Order the elements of _names_ so they are in the same relative order as would be produced by the Iterator that would be returned if the [[Enumerate]] internal method was invoked on _O_.
+        1. Order the elements of _names_ so they are in the same relative order as would be produced by the Iterator that would be returned if the EnumerateObjectProperties internal method was invoked with _O_.
         1. Return _names_.
       </emu-alg>
       <emu-note>
```

Comment:
Is the note on the next line (`"<p>The order of elements in the returned list is the same as the enumeration order that is used by a for-in statement.</p>"` but github has trouble showing it and won't let me comment on it) still necessary? It seems implied by `EnumerateObjectProperties`.